catalog/plugin/catalog-plugin-metacard-validation/pom.xml

```diff
@@ -84,7 +84,7 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.9</minimum>
+                                            <minimum>0.8</minimum>
```

Comment:
According to a quick verify run the covered ratios can actually be set to 0.88/0.90/0.85/0.88 (instruction/branch/complexity/line).
Also groupId (line 25) is coming up as redundant/unnecessary (remove?)